frontend/src/app/components/MainInstallButton.js

```diff
@@ -31,13 +35,16 @@ export default class MainInstallButton extends React.Component {
   }
 
   renderInstallButton(isInstalling, hasAddon) {
+    let { experimentTitle } = this.props;
     return (
       <div>
         <button onClick={e => this.install(e)}
                 className={classnames('button extra-large primary install', { 'state-change': isInstalling })}>
           {hasAddon && <span className="progress-btn-msg" data-l10n-id="landingInstalledButton">Installed</span>}
           {!hasAddon && !isInstalling &&
-            <span className="default-btn-msg" data-l10n-id="landingInstallButton">Install the Test Pilot Add-on</span>}
+            <span>
+              <span className="default-btn-msg"><span data-l10n-id="landingInstallButton">Install the Test Pilot Add-on</span> { experimentTitle && "+" } { experimentTitle }</span>
```

Comment:
This is an amazing hack that I came up with. I changed the button text to "Install the Test Pilot Add-on + Experiment Name" which doesn't require any localization changes. However, we probably want to localize this differently, instead.